README.md

```diff
@@ -19,3 +42,10 @@ or add this into package.json
       "**/babel-preset-react-app/@babel/runtime"
 ]
 ```
+
+## Contributing
+Please read [CONTRIBUTING.md](https://github.com/oraichain/oraiswap-frontend/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## License
+
+Released under the [Apache 2.0 License](https://github.com/oraichain/oraiswap-frontend/LICENSE).
```

Comment:
Here too, I think we can use relative path instead of an absolute URL

Reply:
Thanks for your contribution. I will change it